lib/Net/SAML2/SP.pm

```diff
@@ -291,6 +291,12 @@ sub authn_request {
         issuer        => $self->id,
         destination   => $_[0],
         nameid_format => $_[1],
+        NonEmptySimpleStr->check($_[2])
```

Comment:
This means you can inject any word in here where we only want `1|0|true|false` for booleans. I don't think we should do that.

Reply:
Good point